controllers/terminal/README.md

```diff
@@ -27,4 +34,9 @@ TerminalSpec
 ## Usage
 1. run `kubectl apply terminal.yaml`
 2. run `kubectl get service serviceName` to get the nodePort that the service expose. The serviceName is same with the terminal name.
-3. Visit http://ip:port
\ No newline at end of file
+3. Visit http://ip:port
```

Comment:
write to status is better ?

Reply:
Only write port to status?

Reply:
can write domain dns , ex  name.default.cluster.local:8080    @fanux  Is this necessary here?

Reply:
We will create a applications object later, the UI endpoint can set into applications spec feiled.

Reply:
After Disscusion, we could add auto support of host like terminal-UID-Random.cloud.sealos.io into ingress/nginx.